### PR TITLE
Use longer driver timeouts during tests

### PR DESCRIPTION
**Pull request by dimas-b**
* CI is very slow

### CODE REVIEW

**Review by EricBorczuk — APPROVED** [review]
Is this why my tests keep failing due to timeouts xD (https://github.com/stargate/stargate/pull/211/checks?check_run_id=1218259317)

File: testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java [hunk 1]
```diff
@@ -61,8 +61,9 @@ protected CqlSessionBuilder newSessionBuilder() {
     config.put(
         TypedDriverOption.LOAD_BALANCING_POLICY_CLASS,
         DcInferringLoadBalancingPolicy.class.getName());
-    config.put(TypedDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(60));
-    config.put(TypedDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(30));
+    config.put(TypedDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(180));
+    config.put(TypedDriverOption.METADATA_SCHEMA_REQUEST_TIMEOUT, Duration.ofSeconds(180));
+    config.put(TypedDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofSeconds(180));
     config.put(TypedDriverOption.REQUEST_TRACE_INTERVAL, Duration.ofSeconds(5));
```
Comment (tomekl007) [inline testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java:67]: you may consider increasing also:
`CONNECTION_INIT_QUERY_TIMEOUT("advanced.connection.init-query-timeout"),`

**Review by jorgebay — APPROVED** [review]
oh my...

Thanks for taking care of this.

I think we should move out of GH actions for CI.

**Review by mpenick — APPROVED** [review]

File: testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java (hunk 1 above)
Comment (dimas-b) [reply testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java:67]: I'll probably do another PR is case tests timeout again... do not want to lose reviews and clean CI run :)

**Pull request merged**